CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 - Fixed connectors not showing on operator's "All connectors" list ([#119](https://github.com/sovity/authority-portal/issues/119))
 - Changed Access Token Lifespan to be one minute ([#111](https://github.com/sovity/authority-portal/issues/111))
 - Fixed an issue with connector URLs not being validated correctly in the UI ([#115](https://github.com/sovity/authority-portal/issues/115))
+- Add tooltip with info about participant roles ([#76](https://github.com/sovity/authority-portal/issues/76))
```

Comment:
Since we're not talking only about participant roles but also about application roles, how about:
"Added tooltips explaining user roles"

Reply:
done